python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ethernet-interfaces.j2

```diff
@@ -822,6 +822,93 @@ interface {{ ethernet_interface.name }}
 {%     if ethernet_interface.isis_authentication_key is arista.avd.defined %}
    isis authentication key 7 {{ ethernet_interface.isis_authentication_key | arista.avd.hide_passwords(hide_passwords) }}
 {%     endif %}
+{%     if ethernet_interface.isis_authentication.both.mode is arista.avd.defined
```

Comment:
can we add this condition before to if ethernet_interface.isis_authentication_mode and keep the if ethernet_interface.isis_authentication_mode in elif so in this way we will priorities the new data model plus avoid to have duplicate config in case both type of input is given. 
Same for authentication key

Reply:
Fixed.